pkg/apis/batch/validation/validation.go

```diff
@@ -743,11 +743,7 @@ func validateCronJobSpec(spec, oldSpec *batch.CronJobSpec, fldPath *field.Path,
 	if len(spec.Schedule) == 0 {
 		allErrs = append(allErrs, field.Required(fldPath.Child("schedule"), ""))
 	} else {
-		allowTZInSchedule := false
```

Comment:
I wasn't expecting to drop this ratcheting validation ever... removing this toleration makes unrelated updates to cronjobs containing a TZ (like annotating them or removing a finalizer) fail

Reply:
We're already requiring use of the field in new cronjobs, I don't see a reason to break updates of old cronjobs